tsconfig.json

```diff
@@ -7,10 +7,9 @@
         "moduleResolution": "node",
         "rootDir": "./",
         "esModuleInterop": true,
-        // "noImplicitAny": false
```

Comment:
why did you commented it? Looks like you don't have `any` types yet

Reply:
fixed